scss/components/_utils.scss

```diff
@@ -2,3 +2,25 @@
   position: relative;
   top: -1px;
 }
+
+.civicase__3-dot-menu {
+  .btn {
+    border: 0;
+    height: 18px;
+    margin-left: 6px !important;
```

Comment:
I'm not too familiar with the markup, but can `!important` be avoided at all? From my experience once you start adding `!important` everything goes down hill from there and needs to be added everywhere. Avoid it if possible (if possible).